### ISSUE TITLE
Possible to select city in left sidebar in Sidewalk Gallery?

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#2633]
Ideally, Sidewalk Gallery would provide a unified view of all data collect from Project Sidewalk across all deployment cities and "City" would just be another filterable element on the left sidebar.

However, perhaps in the short-term we could at least add 'City' to the dropdown—this would at least create more unification across cities in the tool? Then, when they select the "new" city, we would reload the page on that city's server. Of course, the user might expect that their selected filters would also automatically apply (so we might have to pass that via REST or something).

**Comment by misaugstad** [comment]
I like it! Linking to the page in another city would be easy. Including the filters would (I think/hope) not be terribly difficult either!

**Comment by misaugstad** [comment]
@sidlak-c137 you can see the list of cities (well, city IDs) at the top of the conf/cityparams.conf file, and these can be accessed programmatically (there are examples of that throughout the code, but lmk if you need help with it). The URLs for each city are also in that config file. The links to other cities on the landing page are one example to go off of.

The extra thing I'd like you to do here is make a new config in the config file, we might call it "status". For now you can make the status of Seattle, Newberg, SPGG, CDMX, and Columbus "public, and that status of DC, Pittsburgh, and Chicago can be "private". Then you can use that config to programmatically show only the "public" cities in the Gallery dropdown. And while you're doing that, updating the list of links on the landing page to use that instead of having the hard-coded filtering of the 3 private ones would be great :)

In the future, we can adapt this config or add more to them, but we should just get it started now!

**Comment by sidlak-c137** [comment]
Just making sure, do each of the cities have the same filter-by tags for each filter option in the gallery?

**Comment by jonfroehlich** [comment]
Ugh, good point. They are _mostly_ the same but some there are slight distinctions based on our partner preferences and geographic contexts. @misaugstad could tell you more.

**Comment by misaugstad** [comment]
Ahh right. As of right now, the only difference is in the tags. You can see the differences in the `cityparams.conf` file, looking at the `excluded-tags` parameter. The gist is that (as of right now) US cities do not have "garage entrance", "stairs", or "street vendor" tags, while Mexico cities do not have the "fire hydrant", "parked bike", "construction", or "brick" tags.

**Comment by sidlak-c137** [comment]
I thought I'd give an update on this issue. I have made it so that there is a dropdown that allows the user to change the city. Choosing a different city in the dropdown will redirect the user to the selected city's server's gallery page. Here is a video of it in action:

https://user-images.githubusercontent.com/15069663/150877994-7f3efc54-d54e-4b2d-adec-e48bd74f163a.mp4

I am currently trying to get the filters to move over to the new server, but I got the basic server-redirect functionality working. Also, the cities that appear in the dropdown are only cities that are "public" and the one that you are currently in (ex, chicago is not public, but it is what I was using for testing). The dropdown also defaults to the current city.
